src/styles/styles.ts

```diff
@@ -2159,7 +2159,6 @@ const styles = (theme: ThemeDefault) =>
             // It's being used on Web/Desktop only to vertically center short PDFs,
             // while preventing the overflow of the top of long PDF files.
             ...display.dGrid,
-            backgroundColor: theme.modalBackground,
```

Comment:
Removed this one here, it's only used in the touched component on desktop, and does not really add anything. This was the one causing the weird background on hover.